### PR TITLE
Test pr

### PR DESCRIPTION
**Pull request by lolero**
I added some test files and updated the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+
+Runs the app in the development mode.
+Open http://localhost:3000 to view it in the browser.
+
+The page will reload if you make edits.
+You will also see any lint errors in the console.
+
+yarn test
+
+Launches the test runner in the interactive watch mode.
```
Comment (gbogner) [inline README.md:17]: this could be done in a better way

File: src/ColumnPaginatorComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+interface LocalState {
+}
+
+interface LocalProps {
+  matrixId: ViewId;
+}
+
+type StyledLocalProps = LocalProps & WithStyles<typeof styles>;
+
+const styles = (theme:Theme):StyleRules => createStyles({
```
Comment (gbogner) [inline src/ColumnPaginatorComponent.tsx]:
```suggestion
const stylesss = (theme:Theme):StyleRules => createStyles({
```

**Review by gbogner — CHANGES_REQUESTED** [review]
looks good but i made a few comments and suggestions

File: README.md (hunk 1 above)
Comment (lolero) [reply README.md:17]: thanks will fix

**Review by gbogner — APPROVED** [review]
👍

**Pull request merged**